BrainPortal/app/views/layouts/_neurohub_flash_message.html.erb

```diff
@@ -35,3 +35,16 @@
       <%= (h(msg.to_s.strip).gsub(/\n/, '<BR/>')).html_safe %>
     </div>
   <%- end -%>
+
+  <div class="card">
+  <%- unless @display_messages.blank? %>
+
+    <%- for mess in @display_messages -%>
+      <p>
+      <div class="nh-invitations card-item">
+      <%= render(:partial  => "nh_invitations/message_details", :locals  => { :mess  => mess } ) rescue "Oh oh. your message was not rendered. Talk to devs!" %>
```

Comment:
You do not need a rescue on a render. Rails will handle that.

In meantime we have the same code in `./messages/_message_display.html.erb` ==> ligne 33.

Reply:
well, much code is similiar to similiar cbrain features. Do you say I should 'unrescue' old code too?

Reply:
Can you give me example in CBRAIN ? I did not see them.

Reply:
./messages/_message_display.html.erb ==> ligne 33.

Reply:
Yep, I already mention it. Seems to be the 2 only place with this strange rescue. Please remove at both place.

Reply:
besides the templates, much of message processing in neurohub_controller.rb is copy of cbrain_controller.

Reply:
It does not handle well errors on my pc ![image](https://user-images.githubusercontent.com/11949081/90425669-1de8df00-e08e-11ea-83c3-b13676a81a95.png)

Reply:
Try in the Rails `production` environment

Reply:
in production mode during malfunction the message seems silently hidden, which mean the user will do not know that his message got lost. Do you still want remove that rescues ? @natacha-beck

Reply:
Just use the same `rescue` that was there on the CBRAIN side, then.

Reply:
I don't want to waste too much time about this anymore.

Reply:
As a compromise I removed the rescue form neurohub as users might understand that it is experimental project and confirmations could be lost